spec/stat_tracker_spec.rb

```diff
@@ -6,9 +6,9 @@
 
   before :each do
 
-    @game_path = './data/games_dummy_revised.csv'
+    @game_path = './data/games_dummy.csv'
     @team_path = './data/teams.csv'
-    @game_teams_path = './data/game_teams_dummy_revised.csv'
+    @game_teams_path = './data/game_teams_dummy.csv'
 
```

Comment:
Is this change switching the file paths back to the original dummy files?

Reply:
good catch - i did a pull from main before working on this and I'm not sure why this didn't update for me... I am a little concerned other things might not me up to date.